CMake/cdat_modules_extra/setup_runtime.csh.in

```diff
@@ -22,11 +22,12 @@ endif
 # which can lead to errors.
 if ( $?UVCDAT_SETUP_PATH ) then
   if ( ${UVCDAT_SETUP_PATH} == ${install_prefix} ) then
-    echo 'UVCDAT setup already sourced for this install location.'
+    echo 'Nothing to do UVCDAT is already setup at '${UVCDAT_SETUP_PATH}
```

Comment:
How about:
'Nothing to do since UVCDAT is already setup at '${UVCDAT_SETUP_PATH}